### DISCUSSION TITLE
Using oh-my-posh theme when connecting to remote server via ssh

### DISCUSSION THREAD

**Discussion by ranggasan** [JanDeDobbeleer/oh-my-posh#3524] [General]
I have windows computer where the OMP is installed and I'd like to remote linux server via SSH. Is there anyway to keep using OMP theme when we remote server via SSH?

**Comment by JanDeDobbeleer** [comment]
@ranggasan only by installing on that computer aa well.

**Reply by ranggasan** [reply]
Oh, I see. Thank you
